ephypype/gather/gather_results.py

```diff
@@ -0,0 +1,266 @@
+                        subjects_folder, psd_node)
+
+    for sbj in subject_id:
+        for ses in session_id:
+            file_path = node_path.format(ses=ses, sbj=sbj)
+            psd_file = glob.glob(op.join(file_path, '*.npz'))
+            coo_file = glob.glob(op.join(file_path, '*coords.txt'))
+            if psd_file:
+                psd_files.append(psd_file[0])
+            if coo_file:
+                channel_coo_files.append(coo_file[0])
+
+    return psd_files, channel_coo_files
+
+
+def get_inverse_files(workflow_path, workflow_name, subject_id, session_id):
+    """
+    Get source reconstruction files.
+
+    Parameters
+    ----------
+       workflow_path : str
+           Path of connectivity workflow
+       workflow_name : str
+           Name of the connectivity workflow
+       subject_id : list of str
+           List of subjects id
+       session_id : list of str
+           List of sessions
+
+    Returns
+    -------
+        time_series_file : list of str
+            List of path of source reconstruction files
+        label_files : list of str
+            List of path of labels pickle file
+    """
+    inverse_pipeline = 'inv_sol_pipeline'
+    inverse_node = 'inv_solution'
+
+    time_series_files = list()
+    label_files = list()
+
+    subjects_folder = '_session_id_{ses}_subject_id_{sbj}'
+    node_path = op.join(workflow_path, workflow_name, inverse_pipeline,
+                        subjects_folder, inverse_node)
+
+    for sbj in subject_id:
+        for ses in session_id:
+            file_path = node_path.format(ses=ses, sbj=sbj)
+            inverse_file = glob.glob(op.join(file_path, '*.npy'))
+            label_file = glob.glob(op.join(file_path, '*.pkl'))
+            if inverse_file:
+                time_series_files.append(inverse_file[0])
+            if label_file:
+                label_files.append(label_file[0])
+
+    return time_series_files, label_files
+
+
+def get_ica_solution(workflow_path, workflow_name, subject_id, session_id):
+    """
+    Get ica solution.
+
+    Parameters
+    ----------
+       workflow_path : str
+           Path of connectivity workflow
+       workflow_name : str
+           Name of the connectivity workflow
+       subject_id : list of str
+           List of subjects id
+       session_id : list of str
+           List of sessions
+
+    Returns
+    -------
+        ica_solution_file : list of str
+            List of path of source reconstruction files
+        label_files : list of str
+            List of path of labels pickle file
+    """
+    ica_pipeline = 'preproc_meeg_pipeline'
+    ica_node = 'ica'
+
+    ica_files = list()
+    raw_files = list()
+
+    subjects_folder = '_session_id_{ses}_subject_id_{sbj}'
+    node_path = op.join(workflow_path, workflow_name, ica_pipeline,
+                        subjects_folder, ica_node)
+
+    for sbj in subject_id:
+        for ses in session_id:
```

Comment:
> > @EtienneCmb why? in my laptop it works with visbrain master branch
> 
> @annapasca it's not working because it's on the master branch but I didn't publish a release on pip. You've to clone the visbrain repo but you can stay on the master branch

graziieeeeeee!!!! :) now I understand the difference :) I got crazy yesterday jajaja